src/app/components/vm-list/vm-list.component.html

```diff
@@ -85,6 +85,20 @@
           >
             Shutdown
           </button>
+          <button
+            mat-menu-item
+            (click)="openSelectedHere()"
+            [disabled]="selectedVms.length === 0"  
+          >
+            Open in Player tab
```

Comment:
Not the most important, but just wanted to ask about naming here.  "VM" would be weird since technically a View Creator could call the VM application whatever they want.  I'm fine with it as is, but wanted to see if there was any better suggestion @sei-aschlackman

Reply:
Hmm, didn't think of that. It is the Player Vm App, not like it can really stand alone, so it doesn't bother me much. Otherwise, I would suggest something like "Open in application tab"? @sei-rlehman have any thoughts?

Reply:
I based the button naming off of @sei-rlehman 's comment on the ticket: "I think adding "Open in Player tab" and "Open in browser tab" to the Actions button makes sense."